src/test/java/org/jabref/logic/citationstyle/CitationStyleGeneratorTest.java

```diff
@@ -58,6 +60,23 @@ void aPACitation() {
         assertEquals(expected, citation);
     }
 
+    /**
+     * Fails due to citeproc-java ({@link CitationStyleGenerator#generateInText(List, String, CitationStyleOutputFormat, BibDatabaseContext, BibEntryTypesManager) generateInText}) returning an empty citation.
+     * Alphanumeric citations are thus, currently manually generated by formatting (see {@link org.jabref.logic.openoffice.oocsltext.CSLFormatUtils#generateAlphanumericCitation(List, BibDatabaseContext) generateAlphaNumericCitation}).
+     */
+    @Test
+    @Disabled
+    void din1502AlphanumericInTextCitation() throws IOException {
+        context.setMode(BibDatabaseMode.BIBLATEX);
+        CitationStyle style = styleList.stream().filter(e -> "DIN 1505-2 (alphanumeric, Deutsch) - standard superseded by ISO-690".equals(e.getTitle())).findAny().get();
+        Citation citation = CitationStyleGenerator.generateInText(List.of(testEntry), style.getSource(), CitationStyleOutputFormat.HTML, context, bibEntryTypesManager);
+        String inTextCitationText = citation.getText();
+
+        String expected = "[Smit2016]";
+
+        assertEquals(expected, inTextCitationText);
```

Comment:
One-liners can be inlined

```suggestion
        assertEquals("[Smit2016]", inTextCitationText);
```